leapp/actors/__init__.py

```diff
@@ -470,6 +482,8 @@ def get_actor_metadata(actor):
         _get_attribute(actor, 'dialogs', _is_dialog_tuple, required=False, default_value=()),
         _get_attribute(actor, 'description', _is_type(string_types), required=False,
                        default_value=actor.__doc__ or 'There has been no description provided for this actor.'),
+        _get_attribute(actor, 'config_schemas', _is_type(Config), required=False,
+                       default_value=actor.__doc__ or 'Description of the configuration used by this actor.'),
```

Comment:
We will want here something similar like ``_is_config_tuple`` similar to the functions above. That is if I understand it correctly and we are allowing multiple config schema definitions per actor. Am I right @abadger ?

Reply:
Nice catch!  I'ce implemented such a function.  The one change I made from how _is_model_tuple() and etc operate is to check for a Sequence instead of a Tuple.  This allows Tuples, Lists, NamedTuples, to all satisfy the requirement rather than only tuples.  If that shouldn't be the case, we can change it back.